cmd/kueuectl/app/list/list_localqueue.go

```diff
@@ -0,0 +1,170 @@
+being active/inactive, belonging to the specified namespace, matching 
+the label selector or the field selector.`
+	lqExample = `  # List LocalQueue 
+  kueuectl list localqueue`
+)
+
+type LocalQueueOptions struct {
+	PrintFlags *genericclioptions.PrintFlags
+
+	AllNamespaces      bool
+	EnforceNamespace   bool
+	Namespace          string
+	FieldSelector      string
+	LabelSelector      string
+	ClusterQueueFilter string
+
+	Client kueuev1beta1.KueueV1beta1Interface
+
+	PrintObj printers.ResourcePrinterFunc
+
+	genericiooptions.IOStreams
+}
+
+func NewLocalQueueOptions(streams genericiooptions.IOStreams) *LocalQueueOptions {
+	return &LocalQueueOptions{
+		PrintFlags: genericclioptions.NewPrintFlags("").WithTypeSetter(scheme.Scheme),
+		IOStreams:  streams,
+	}
+}
+
+func NewLocalQueueCmd(clientGetter genericclioptions.RESTClientGetter, streams genericiooptions.IOStreams) *cobra.Command {
+	o := NewLocalQueueOptions(streams)
+
+	cmd := &cobra.Command{
+		Use: "localqueue [-–clusterqueue CLUSTER_QUEUE_NAME] [--selector key1=value1] [--field-selector key1=value1] [--all-namespaces]",
+		// To do not add "[flags]" suffix on the end of usage line
+		DisableFlagsInUseLine: true,
+		Aliases:               []string{"lq"},
+		Short:                 "List LocalQueue",
+		Long:                  lqLong,
+		Example:               lqExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			cobra.CheckErr(o.Complete(clientGetter, cmd, args))
+			cobra.CheckErr(o.Run(cmd.Context()))
+		},
+	}
+
+	o.PrintFlags.AddFlags(cmd)
+
+	addAllNamespacesFlagVar(cmd, &o.AllNamespaces)
+	addFieldSelectorFlagVar(cmd, &o.FieldSelector)
+	addLabelSelectorFlagVar(cmd, &o.LabelSelector)
+	addClusterQueueFilterFlagVar(cmd, &o.ClusterQueueFilter)
+
+	return cmd
+}
+
+// Complete completes all the required options
+func (o *LocalQueueOptions) Complete(clientGetter genericclioptions.RESTClientGetter, cmd *cobra.Command, args []string) error {
+	var err error
+
+	o.Namespace, o.EnforceNamespace, err = clientGetter.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		return err
+	}
+
+	config, err := clientGetter.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	clientset, err := versioned.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	o.Client = clientset.KueueV1beta1()
+
+	if !o.PrintFlags.OutputFlagSpecified() {
+		printer := newLocalQueueTablePrinter()
+		if o.AllNamespaces {
+			printer.WithNamespace()
+		}
+		o.PrintObj = printer.PrintObj
+	} else {
+		printer, err := o.PrintFlags.ToPrinter()
+		if err != nil {
+			return err
+		}
+		o.PrintObj = printer.PrintObj
+	}
+
+	return nil
+}
+
+// Run performs the list operation.
+func (o *LocalQueueOptions) Run(ctx context.Context) error {
+	namespace := o.Namespace
+	if o.AllNamespaces {
+		namespace = ""
+	}
+
+	opts := metav1.ListOptions{LabelSelector: o.LabelSelector, FieldSelector: o.FieldSelector}
+	list, err := o.Client.LocalQueues(namespace).List(ctx, opts)
```

Comment:
Maybe not necessary for LocalQueues, but we should think about paging for huge lists.

Reply:
Yes, and maybe about watch.

Reply:
Please leave an issue for supporting paging